service/create/policy.go

```diff
@@ -63,7 +64,11 @@ func createRole(svc *iam.IAM, kmsKeyArn, s3Bucket, clusterID string) error {
 		RoleName:                 aws.String(clusterRoleName),
 		AssumeRolePolicyDocument: aws.String(AssumeRolePolicyDocument),
 	}); err != nil {
-		return err
+		if aerr, ok := err.(awserr.Error); ok && aerr.Code() == iam.ErrCodeEntityAlreadyExistsException {
+			s.logger.Log("info", fmt.Sprintf("role '%s' exists, reusing", clusterRoleName))
+		} else {
+			return err
```

Comment:
We tend to use error masking with the microkit error library. This makes debugging more easy when shit hits the fan.